.github/workflows/upload-and-deploy.yaml

```diff
@@ -14,6 +14,7 @@ env:
   PROD_INPUT_BUCKET: recover-input-data
   PROD_INTERMEDIATE_BUCKET: recover-intermediate-data
   INTEGRATION_TEST_NUM_EXPORTS: 28
+  ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
```

Comment:
Hmm. Does it make sense to update to newer versions or is that a harder lift?

Reply:
So the problem is the aws glue docker containers (both glue 4.0 and 3.0) haven't been updated to use node 20 hence they all have node 16 / the glibc version mismatch.

The above would only be a temp. solution though. A more long term solution (until the above gets fixed) i was thinking of and mentioned in the discussion is by building Node.js statically so it doesn't matter what glibc version is used by the docker container. Not sure how viable this is yet so going to try it.